src/python/pants/backend/python/subsystems/pex_build_util.py

```diff
@@ -140,40 +144,57 @@ def subsystem_dependencies(cls):
         PythonSetup,
       )
 
+    @property
+    def generate_ptex(self) -> bool:
+      return cast(bool, self.get_options().generate_ptex)
+
     @classmethod
-    def create(cls, builder, log=None):
+    def create(cls, builder, log=None, generate_ptex=None, pex_requirement=None):
       options = cls.global_instance().get_options()
       setuptools_requirement = f'setuptools=={options.setuptools_version}'
+      pex_requirement = f'pex=={pex_requirement}' if pex_requirement else 'pex'
```

Comment:
Seems strange that if `pex_requirement` is `None`, we still have a `pex_requirement`? I don't entirely understand this... Are we saying that all pexes now have a requirement of `pex`, but sometimes it floats?